rclpy/rclpy/qos.py

```diff
@@ -106,18 +106,36 @@ def get_c_qos_profile(self):
 
 
 class QoSHistoryPolicy(IntEnum):
+    """
+    Class defining enums for QoS History settings.
```

Comment:
Updated in https://github.com/ros2/rclpy/pull/49/commits/fd1e9b6597cefae1518723223a3a26e0b7683279